src/pages/register.astro

```diff
@@ -2,5 +2,5 @@
 import Layout from '../layouts/Layout.astro'
 import VueRegister from '../vue-pages/register.vue'
 ---
-
+<script is:inline src="https://challenges.cloudflare.com/turnstile/v0/api.js" async defer></script> 
```

Comment:
Since the `Layout ` declares the HTML, I'd recommend we shift this to the `Layout` component to be initialized in the `head` tag with a flag of if the current URL is `/register`.

Reply:
I believe that what I did is not wrong. I followed the [official documentation](https://docs.astro.build/en/guides/client-side-scripts/)

Reply:
- my intention is not to say that you did something wrong and neither to discourage you
- what I am saying is that the script is:inline is fine, but it can be included Layout.astro file like https://github.com/Cap-go/website/blob/main/src/layouts/Layout.astro#L26-L37 where you are including this script in head based on the current URL

Reply:
Frankly, I am against this change. I believe that although providing the script in the `head` is the standard in HTML, in this case it makes more sense to keep this in `register.astro` so that is easily findable. 

Layout.astro provides general scripts that are always loaded, and even it doesn't follow the HTML standard of putting scripts in the `head` tag.

Reply:
I prefer as well to not load turnstile for users just visiting landing.